test/UniStaker.t.sol

```diff
@@ -4905,6 +4905,33 @@ contract ClaimReward is UniStakerRewardsTest {
     assertEq(rewardToken.balanceOf(_depositor), _earned);
   }
 
+  function testFuzz_ReturnsRewardAmount(
```

Comment:
Maybe rename to `testFuzz_ReturnsClaimedRewardAmount` like the others

Reply:
@alexkeating  changed in 1c62cd2